src/resource/execution/args.rs

```diff
@@ -45,6 +49,11 @@ impl Args {
         self.script = Some(id);
     }
 
+    /// Set the name to execute.
+    pub fn set_name(&mut self, name: String) {
```

Comment:
Since this is just a handy convenience wrapper anyway, let's make it extra convenient and use the fancy version:

```rust
pub fn set_name<S>(&mut self, name: S)
where
    S: Into<String>,
{
    self.name = Some(name.into());
}
```

This allows passing in `String`, `&str`, etc. Since this function has no real good reason for existing, let's ask it add an extra smidgen of convenience at least.